lib/query.dart

```diff
@@ -43,4 +46,29 @@ class Query {
 
   static String parseValues(dynamic value) =>
       (value is String) ? '"$value"' : '$value';
+
+  String method;
+  List<dynamic> params;
```

Comment:
```suggestion
  final String method;
  final List<dynamic> params;
```
I think these can be final